gtest/src/js/mod.rs

```diff
@@ -168,20 +168,20 @@ mod tests {
 
         let json = MetaData::Json(json);
 
-        let bytes = json.clone().convert(
-            "examples/target/wasm32-unknown-unknown/release/demo_meta.meta.wasm",
-            &MetaType::Input,
-        );
-        let bytes = bytes.unwrap_or(
-            json.convert(
-                "target/wasm32-unknown-unknown/release/demo_meta.meta.wasm",
+        let bytes = json
+            .clone()
+            .convert(
+                "examples/target/wasm32-unknown-unknown/release/demo_meta.meta.wasm",
                 &MetaType::Input,
             )
-            .expect("Could not find file "),
-        );
+            .or(json.convert(
+                "target/wasm32-unknown-unknown/release/demo_meta.meta.wasm",
+                &MetaType::Input,
+            ));
```

Comment:
It is not recommended to call expensive functions in the `.or()` method. Use the lazy alternative.
```suggestion
            .or_else(|_| json.convert(
                "target/wasm32-unknown-unknown/release/demo_meta.meta.wasm",
                &MetaType::Input,
            ));
```

Reply:
Done